bundler/cli.js

```diff
@@ -24,26 +25,36 @@ function add(value) {
 	command = null
 }
 
+function format(n) {
+	return n.toString().replace(/(\d)(?=(\d\d\d)+(?!\d))/g, "$1,")
+}
+
 bundle(params.input, params.output, {watch: params.watch})
 if (params.minify) {
-	minify(params.output, params.output, {watch: params.watch, advanced: params.aggressive}, function(stats) {
-		function format(n) {
-			return n.toString().replace(/(\d)(?=(\d\d\d)+(?!\d))/g, "$1,")
-		}
-
-		console.log("Original size: " + format(stats.originalGzipSize) + " bytes gzipped (" + format(stats.originalSize) + " bytes uncompressed)")
-		console.log("Compiled size: " + format(stats.compressedGzipSize) + " bytes gzipped (" + format(stats.compressedSize) + " bytes uncompressed)")
-
-		if (params.save) {
-			var readme = fs.readFileSync("./README.md", "utf8")
-			var kb = stats.compressedGzipSize / 1000
-
-			fs.writeFileSync("./README.md",
-				readme.replace(
-					/(<!-- size -->)(.+?)(<!-- \/size -->)/,
-					"$1" + (kb % 1 ? kb.toFixed(2) : kb) + " KB$3"
+	try {
+		// mFiles = { original: String(mithril.js), compressed: String(mithril.min.js) }
+		var mFiles = minify(params.output, {watch: params.watch})
+
+		if (mFiles !== null) {
+			var originalSize = mFiles.original.length
+			var compressedSize = mFiles.compressed.length
+			var originalGzipSize = zlib.gzipSync(mFiles.original).byteLength
+			var compressedGzipSize = zlib.gzipSync(mFiles.compressed).byteLength
+		
+			console.log("Original size: " + format(originalGzipSize) + " bytes gzipped (" + format(originalSize) + " bytes uncompressed)")
+			console.log("Compiled size: " + format(compressedGzipSize) + " bytes gzipped (" + format(compressedSize) + " bytes uncompressed)")
+
+			if (params.save) {
+				var readme = fs.readFileSync("./README.md", "utf8")
+				var kb = compressedGzipSize / 1000
+
+				fs.writeFileSync("./README.md",
+					readme.replace(
+						/(<!-- size -->)(.+?)(<!-- \/size -->)/,
+						"$1" + (kb % 1 ? kb.toFixed(2) : kb) + " KB$3"
+					)
 				)
-			)
+			}
 		}
-	})
-}
+	} catch(e) { console.error(e) }
```

Comment:
I think you can let the error go to the cli-script and terminate the bundling process.
@isiahmeadows would you do that?

Reply:
@StephanHoyer files are already bundled at this point, so we'd only be terminating minification, if that affects your thinking.

Reply:
@StephanHoyer @isiahmeadows This behavior is consistent with the existing story, meaning if Closure Compiler chokes today we'd still get an un-minified `mithril.min.js` in the build. While we're in here, do we want to crack open `bundle.js` and re-sequence things? Now that we're local and sync, it seems sane.

Reply:
@StephanHoyer And yes, I'd rather propagate errors to the top level instead. We shouldn't be generating syntactically invalid bundles - we already check for syntactic validity when linting, and if things break, it implies either a bug in our stuff or in a dev dependency. I'm not really a fan of masking bugs like that, so let's just remove the `try`/`catch` block for now @CreaturesInUnitards.

Reply:
alrighty then! :)